sysutils/sshpass/Portfile

```
@@ -3,11 +3,11 @@
 PortSystem          1.0
 
 name                sshpass
-version             1.09
+version             1.10
 categories          sysutils
 license             GPL-2
 platforms           darwin
-maintainers         nomaintainer
+maintainers         @manakbisht
```

Thanks for volunteering to be a maintainer for this port.  That said, a port should also be marked `openmaintainer` in general, which means that others besides you are allowed to make minor updates to this port.

```suggestion
maintainers         @manakbisht openmaintainer
```